sda-commons-server-jackson/src/test/java/org/sdase/commons/server/jackson/test/NameNotRepeatedValidator.java

```diff
@@ -19,13 +19,11 @@ public boolean isValid(ValidationResource value, ConstraintValidatorContext cont
       context.disableDefaultConstraintViolation();
       context
           .buildConstraintViolationWithTemplate("First name and last name must be different.")
-          .addPropertyNode("firstName")
-          .addBeanNode()
+          .addPropertyNode("name")
```

Comment:
This custom validator must set the correct (=resolved) name of the property since this is not forwarded to the `PropertyNodeNameProvider`. Since this code is only used for testing, I guess we can just change it.

In addition, the call `addBeanNode()` seems to be wrong here, since it added an empty path entry to the list that led to the generation of the path `name.` instead of `name`.

Reply:
Can we add a comment to avoid confusion?
```suggestion
          // The property firstName is annotated with @JsonProperty("name")
          .addPropertyNode("name")
```